### DISCUSSION TITLE
PCF8574 mit Tasmota

### DISCUSSION THREAD

**Discussion by comtechnik** [arendst/Tasmota#17882] [Q&A]
Hallo zusammen,
ich habe einen PCF8574, Firmeware auf den neuesten Stand.
Ich möchte die 8 Eingänge als Taster verwenden, nur leider wird mir bei geöffneten Taster immer 1 angezeigt und bei geschlossenen Taster 0
Kann ich dies ändern?
In der Konfiguration des PCF8574 kann man zwar Ports invertieren, nur die hat keine Auswirkung darauf. Die Eingänge bleiben immer bei geöffneten Taster auf 1.

Danke für eure Hilfe

**Comment by Noschvie** [comment]
https://tasmota.github.io/docs/PCF8574/

**Reply by comtechnik** [reply]
Sorry aber daraus werde ich nicht schlau.
Hab ich ja schonmal gesehen, aber irgendwie versteh ich das nicht.

**Reply by barbudor** [reply]
Sorry no comprendo
This is a English forum
Please use english
Thanks

**Reply by Noschvie** [reply]
Maybe this https://forum.creationx.de

**Reply by barbudor** [reply]
The Port Invert features is only for Outputs

For Inputs, it directly reports the state of the pin which is 1 (high) when not connected, and most likely 0 when connect to ground
It doesn't matter, just make your rules accordingly
